src/content/docs/distributed-tracing/concepts/introduction-distributed-tracing.mdx

```diff
@@ -0,0 +1,143 @@
+
+Distributed tracing systems collect data as the requests go from one service to another, recording each segment of the journey as a span. These spans contain important details about each segment of the request and are combined into one trace. The completed trace gives you a picture of the entire request.
+
+  ![Diagram showing how spans are generated in a distributed trace](./images/span-diagram.png "span-diagram.png")
+
+  <figcaption>
+    Here is an example of three spans captured from an HTTP request. These spans are sent to New Relic where they are combined into one distributed trace.
+  </figcaption>
+
+## Why it matters
+
+A request might pass through various microservices to reach completion. The microservices or functions could be located in multiple containers, serverless environments, virtual machines, different cloud providers, on-premises, or any combination of these.
+
+For example, let's say that you're in a situation where a slow-running request affects the experience of a set of customers:
+
+* The request is distributed across multiple microservices and serverless functions. 
+* Several different teams own and monitor the various services that are involved in the request.
+* None of the teams have reported any performance issues with their microservices. 
+
+Without a way to view the performance of the entire request across the different services, it’s nearly impossible to pinpoint where and why the high latency is occurring and which team should address the issue. 
+
+## Instrumentation: The key to distributed tracing
+
+Distributed tracing starts with the instrumentation of your services to enable data collection and correlation across the entire distributed system. Instrumention means either manually adding code to services or installing agents that automatically track trace data. 
+
+Many of our New Relic solutions automatically instrument your services for a large number of programming languages and frameworks. You can also use open source tools and open instrumentation standards to instrument your environment. OpenTelemetry, part of the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/), is becoming the one standard for open source instrumentation and telemetry collection. 
+
+## What you can do with distributed tracing and New Relic
+
+After the data is collected, you can visualize it to see service dependencies, performance, and any anomalous events such as errors or unusual latency. Here are some examples of what you can do with your data:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "250px" }}>
+        What you can do
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        [Detect anomalous spans](/docs/apm/distributed-tracing/ui-data/understand-use-distributed-tracing-data#anomalous-spans)
```

Comment:
for me locally, these links about UI were just taking me to the UI doc, without the anchor links working. Not sure why that'd be.

Reply:
@zuluecho9, I corrected these!